### PR TITLE
Travis mac os

### PR DESCRIPTION
**Pull request by pavart**
Fix for #16

### CODE REVIEW

**Comment by pavart** [comment]
Not sure what is proper formatting for `#ifdef` directive.
Please take a look.
@Badrpas @BondarenkoArtur

**Comment by pavart** [comment]
This PR fixes #16

File: Name_Generator_lib/NameGen_Main.cpp [hunk 1]
```diff
@@ -1,9 +1,18 @@
 #define _CRT_SECURE_NO_WARNINGS
 
+#ifdef __APPLE__
+
+#include <stdlib.h>
+#include <malloc/malloc.h>
+
+#elif __linux
```
Comment (Badrpas) [inline Name_Generator_lib/NameGen_Main.cpp]: This looks like it will break Windows build. Using `#else` would be just enough.

**Review by Badrpas — CHANGES_REQUESTED** [review]

File: Name_Generator_lib/NameGen_Main.cpp (hunk 1 above)
Comment (pavart) [reply Name_Generator_lib/NameGen_Main.cpp]: Done

**Review by Badrpas — APPROVED** [review]

**Pull request merged**